docs/oa.md

```diff
@@ -0,0 +1,372 @@
+# OA数据接口
+
+## Models
+
+### `OAApplyEvent` 提交审批申请
+Name|JSON|Type|Doc
+:---|:---|:---|:--
+`CreatorUserID`|`creator_userid`|`string`| 申请人userid，此审批申请将以此员工身份提交，申请人需在应用可见范围内
+`TemplateID`|`template_id`|`string`| 模板id。可在“获取审批申请详情”、“审批状态变化回调通知”中获得，也可在审批模板的模板编辑页面链接中获得。暂不支持通过接口提交[打卡补卡][调班]模板审批单。
+`UseTemplateApprover`|`use_template_approver`|`uint8`| 审批人模式：0-通过接口指定审批人、抄送人（此时approver、notifyer等参数可用）; 1-使用此模板在管理后台设置的审批流程，支持条件审批。默认为0
+`Approver`|`approver`|`[]OAApprover`| 审批流程信息，用于指定审批申请的审批流程，支持单人审批、多人会签、多人或签，可能有多个审批节点，仅use_template_approver为0时生效。
+`Notifier`|`notifyer`|`[]string`| 抄送人节点userid列表，仅use_template_approver为0时生效。
+`NotifyType`|`notify_type`|`*uint8`| 抄送方式：1-提单时抄送（默认值）； 2-单据通过后抄送；3-提单和单据通过后抄送。仅use_template_approver为0时生效。
```

Comment:
`notify_type` is optional param but it must be set `null` or `1`(default value), when set the `0` value, Tencent response error `{ Code: 301025, Msg: "get approval param error:notify_type" }`, to slove this problem i choice `*uint8` as the param type